backend-server/src/main/java/org/dice/ida/vizsuggest/VizSuggestOrchestrator.java

```diff
@@ -0,0 +1,148 @@
+        int index = fileName.lastIndexOf(".");
+        fileName = fileName.substring(0, index);
+        List<Attribute> attributes = new ArrayList<Attribute>();
+        BufferedWriter writer = new BufferedWriter(new FileWriter(getClass().getClassLoader().getResource("metadata/" + datasetName + "/" + tableName) + fileName));
+        for(int i=0;i<data.numAttributes();i++)
+            attributes.add(data.attribute(i));
+        StringBuilder str = new StringBuilder();
+        str.append("Number of Dataset\t"+data.relationName()+"\n");
+        str.append("Number of records\t"+data.numInstances()+"\n");
+        str.append("Number of Attributes\t"+ data.numAttributes()+"\n");
+        str.append("Name\t\tType\tNom\tInt\tReal\tMissingCount\tMissingPercentage\tUniqueCount\tUniquePercentage\tDist"+"\n");
+        int k=0;
+        for (Attribute s:attributes)
+        {
+            AttributeStats as =  data.attributeStats(k);
+            String stat =  as.toString().split("\\n")[1];
+            String fields[] = stat.split("\\s+");
+            if(fields[6].length()<=1)
+            {
+                if(fields[9].length()<=1)
+                    str.append(s.name()+"\t\t"+fields[1]+"\t"+fields[2]+"\t"+fields[3]+"\t"+fields[4]+"\t"+fields[5]+"\t"+fields[7]+"\t"+fields[8]+"\t"+fields[10]+"\t"+fields[11]+"\n");
+                if(fields[9].length()>1)
+                    str.append(s.name()+"\t\t"+fields[1]+"\t"+fields[2]+"\t"+fields[3]+"\t"+fields[4]+"\t"+fields[5]+"\t"+fields[7]+"\t"+fields[8]+"\t"+fields[9].substring(1)+"\t"+fields[10]+"\n");
+            }
+            if(fields[6].length()>1)
+            {
+                if (fields[8].length()<=1)
+                    str.append(s.name()+"\t\t"+fields[1]+"\t"+fields[2]+"\t"+fields[3]+"\t"+fields[4]+"\t"+fields[5]+"\t"+fields[6].substring(1)+"\t"+fields[7]+"\t"+fields[9]+"\t"+fields[10]+"\n");
+                if (fields[8].length()>1)
+                    str.append(s.name()+"\t\t"+fields[1]+"\t"+fields[2]+"\t"+fields[3]+"\t"+fields[4]+"\t"+fields[5]+"\t"+fields[6].substring(1)+"\t"+fields[7]+"\t"+fields[8].substring(1)+"\t"+fields[9]+"\n");
+            }
+            k++;
+        }
+        writer.write(str.toString());
+        writer.close();
+    }
+
+    public void FileReader() throws Exception {
+        File dir = new File(getClass().getClassLoader().getResource("datasets/" + datasetName).getFile());
+        File[] directoryListing = dir.listFiles();
+        if (directoryListing != null) {
+            for (File child : directoryListing) {
+                if (!child.getName().matches(IDAConst.DSMD_FILE_PATTERN)) {
+                    String fileName = child.getName();
+                    CSVLoader loader = new CSVLoader();
+                    loader.setSource(child);
+                    Instances data = loader.getDataSet();
+                    createMetainfo(fileName, data);
+                }
+            }
+        }
+    }*/
```

Comment:
Its not good to have this much commented code. Maybe put it in a different place?